docs/WalletCreationGuide.md

```diff
@@ -0,0 +1,66 @@
+# Wallet Creation Guide
+
+## Introduction
+
+Specter wallets are designed to provide a user-friendly interface around Bitcoin Core, focusing on multisignature setups with airgapped signing devices, though they also support single-signature wallets. It's developed to make the interaction with Bitcoin Core more convenient for the users.
+
+## Types of Wallets
+
+### Single-Sig Wallets
+- These involve one signing device or hardware wallet generating one seed.
+- They are simpler and require only one secure location for seed backup.
+- However, they are less secure against phishing attacks as one compromised seed can lead to lost Bitcoin.
+
+### Multi-Sig Wallets
+- More secure than single-sig, multisig wallets require two or more signing devices to authorize transactions.
```

Comment:
mention the quorum here: "requires X signatures from a total of Y devices. You can decide about X and Y. A common scenario is 2/3"